tests/test-cases/Group23-VIC-Machine-Service/23-01-Metadata.robot

```diff
@@ -16,12 +16,19 @@
 Documentation     Test 23-01 - Version
 Resource          ../../resources/Util.robot
 Resource          ../../resources/Group23-VIC-Machine-Service-Util.robot
-Suite Setup       Start VIC Machine Server
-Suite Teardown    Terminate All Processes    kill=True
+Suite Setup       Setup
```

Comment:
Its better to be more specific when naming setup and teardown keywords.

Reply:
True.  I followed this naming convention since there are other tests that had more inside those keywords.  This way, you can add other future setup/teardown steps in those keywords once these tests are expanded.